server/model/maintenance.js

```diff
@@ -263,9 +267,21 @@ class Maintenance extends BeanModel {
                 };
 
                 // Create Cron
-                this.beanMeta.job = new Cron(this.cron, {
-                    timezone: await this.getTimezone(),
-                }, startEvent);
+                if (this.strategy === "recurring-interval") {
+                    // For recurring-interval, Croner needs to have interval and startAt
+                    const startDate = dayjs(this.startDate);
+                    const [ hour, minute ] = this.startTime.split(":");
+                    const startDateTime = startDate.hour(hour).minute(minute);
+                    this.beanMeta.job = new Cron(this.cron, {
+                        timezone: await this.getTimezone(),
+                        interval: this.interval_day * 24 * 60 * 60,
+                        startAt: startDateTime.toISOString(),
+                    }, startEvent);
+                } else {
+                    this.beanMeta.job = new Cron(this.cron, {
+                        timezone: await this.getTimezone(),
+                    }, startEvent);
+                }
```

Comment:
nit: For these if-statements, I like to pull code which can be shared out, but am fine with your solution.
(feel free to accept/reject the below attached suggestion)

```suggestion
                const startDate = dayjs(this.startDate);
                const [ hour, minute ] = this.startTime.split(":");
                const startDateTime = startDate.hour(hour).minute(minute);
                const cronerOptions = {
                    timezone: await this.getTimezone(),
                    startAt: startDateTime.toISOString(),
                };
                if (this.strategy === "recurring-interval") {
                    cronerOptions.interval = this.interval_day * 24 * 60 * 60;
                }
                this.beanMeta.job = new Cron(this.cron, cronerOptions, startEvent);
```

Reply:
I realize that your example is a little cleaner, but I think I will just leave it because startAt is not used for anything accept for recurring-interval.